services/tables/src/main/java/com/linkedin/openhouse/tables/controller/TablesController.java

```diff
@@ -476,4 +476,36 @@ public ResponseEntity<Void> purgeSoftDeletedTable(
     return new ResponseEntity<>(
         apiResponse.getResponseBody(), apiResponse.getHttpHeaders(), apiResponse.getHttpStatus());
   }
+
+  @Operation(
+      summary = "Restore Soft Deleted Table",
+      description =
+          "Restores a soft deleted table identified by tableId, databaseId, and deletedAtMs",
+      tags = {"Table"})
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "204", description = "Soft Deleted Table RESTORE: NO CONTENT"),
```

Comment:
why 204 status code? should not this be 200?

Reply:
Returns no response. I was thinking about returning the table along with restoreTable but I wasn't sure if the data would be consumed meaningfully